components/ScheduleHome.vue

```diff
@@ -21,6 +32,29 @@
         </div>
       </div>
     </div>
+    <div v-if="false" class="old-schedule">
+      <div v-for="(day, index) in showsByDate" :key="day.id" class="mb-2 scheduleblock">
+        <div class="flex flex-row justify-between dayname">
+          <h4>{{ dayNames[day[0].day-1] }}</h4>
+          <span>{{ format(new Date(index === 0 ? todayDate :
+            tommorrow.setDate(tommorrow.getDate() + 1)) , 'MMMM dd.') }}</span>
+        </div>
+        <div class="dayshows">
+          <div v-for="show in day" :key="show.id" :class="showAirCheck(index, show.name) ? 'dayblock onair' : 'dayblock'">
```

Comment:
This is an awesome solution! So cool that you can evaluate this `showAirCheck()` function for deciding which class to use